web_embedding/README.md

```diff
@@ -0,0 +1,11 @@
+# web_embedding
+
+This directory contains examples of how to embed Flutter in web apps (without iframes):
+
+* **element_embedding_demo**: Modifies the index.html of a flutter app so it is
+  launched in a custom `hostElement`. This is the most basic embedding example.
+* **ng-flutter**: A simple Angular app (and component) that replicates the above
```

Comment:
Is this additional sample in a follow up PR?

Reply:
Yes, I'll do another PR to add that sample. I want to make sure that the commit that adds the sample is co-authored-by Mark.